### ISSUE TITLE
bun update adds caniuse-lite to dependencies

### ISSUE THREAD

**Issue by hateinternet** [browserslist/update-db#42]
Hi @ai 
I'm not sure its important, but it seems that `bun update caniuse-lite` adds package to `dependencies` if it wasn't there.
For example `pnpm up caniuse-lite` or `yarn up -R caniuse-lite` updates lock file only. Bun works differently
Is this expected behaviour? I noticed this because also trying to add bun support :)

found issue - https://github.com/oven-sh/bun/issues/4209

**Comment by ai** [comment]
Yes, this is a problem. Maybe not a blocker for the release, but I will look for that issue. Thanks.

**Comment by vasilii-kovalev** [comment]
The ticket in the Bun repository is now closed, but the original problem is still present.
